pkg/features/kube_features.go

```diff
@@ -645,6 +645,12 @@ const (
 	//
 	// Disables Accelerator Metrics Collected by Kubelet
 	DisableAcceleratorUsageMetrics featuregate.Feature = "DisableAcceleratorUsageMetrics"
+
+	// owner: @andrewsykim @uablrek
+	// alpha: v1.20
+	//
+	// Allows control if NodePorts shall be created for services with type: LoadBalancer
```

Comment:
We probably want to call out the specific field in Service that this is enabling